app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkControllerTest.kt

```diff
@@ -450,39 +398,19 @@ class BookmarkControllerTest {
         every { bookmarkStore.state.guidBackstack } returns listOf(tree.guid)
         every { bookmarkStore.state.tree } returns tree
 
-        var invokePendingDeletionInvoked = false
-        createController(
-            invokePendingDeletion = {
-                invokePendingDeletionInvoked = true
-            }
-        ).handleBackPressed()
-
-        assertTrue(invokePendingDeletionInvoked)
+        createController().handleBackPressed()
 
         verify {
             navController.popBackStack()
         }
     }
 
-    @Test
-    fun `WHEN onSearch is called with BookmarkFragment THEN navigate to BookmarkSearchDialogFragment`() {
-        val controller = createController()
-
-        controller.handleSearch()
-        verify {
-            navController.navigate(
-                BookmarkFragmentDirections.actionBookmarkFragmentToBookmarkSearchDialogFragment()
-            )
-        }
-    }
```

Comment:
Why was this test deleted? I can still see we have this function

Reply:
Brought back, thanks